docs/usage.rst

```diff
@@ -204,8 +204,7 @@ Below is an example of a connection pool usage:
         return app
 
 
-    loop = asyncio.get_event_loop()
-    app = loop.run_until_complete(init_app())
```

Comment:
This is unfortunately incorrect. We'll need to probably update the example to use the latest aiohttp APIs. 

The old code was initializing an event loop, and running the `init_app()` coroutine in that event loop.  Then aiohttp was reusing the same loop in `web.run_app()`.

The new code (your edits) is running `init_app()` in one loop, and using yet another event loop in `web.run_app()`.

It's a bit convoluted and not obvious, but the core reason is that this example uses old aiohttp APIs.  Anyways, please revert this change.

Reply:
> It's a bit convoluted and not obvious, but the core reason is that this example uses old aiohttp APIs. Anyways, please revert this change.

Done, thanks for the explanation.